unlock-app/src/components/interface/buttons/Button.js

```diff
@@ -116,6 +168,70 @@ export const Button = styled.a`
   }
 `
 
+export const MobileToggle = styled.a`
+  cursor: pointer;
+  height: ${props => props.size || ' 48px'};
+  width: ${props => props.size || ' 48px'};
+  display: grid;
+  padding: 0;
+  border: 0;
+
+  > svg {
+    fill: ${props => props.fillColor || 'var(--grey)'};
+    height: ${props => props.size || ' 48px'};
+    width: ${props => props.size || ' 48px'};
+  }
+
+  &:hover {
+    > svg {
+      fill: ${props => props.fillHoverColor || 'var(--link)'};
+    }
+  }
+`
+
+export const MobileButton = styled.a`
```

Comment:
Not a huge fan to have yet another type of button when most of the code is the same as for `LayoutButton`.